pkg/cloudprovider/providers/vsphere/vsphere.go

```diff
@@ -375,14 +375,6 @@ func newControllerNode(cfg VSphereConfig) (*VSphere, error) {
 	if cfg.Global.VCenterPort == "" {
 		cfg.Global.VCenterPort = "443"
 	}
-	if cfg.Global.VMUUID == "" {
-		// This needs root privileges on the host, and will fail otherwise.
-		cfg.Global.VMUUID, err = getvmUUID()
```

Comment:
It looks like other than in one test VSphereConfig.Global.VMMUID is no longer being used. Can we just clean it up?